packages/paypal-commerce-integration/src/paypal-commerce-script-loader.ts

```diff
@@ -127,7 +130,14 @@ export default class PayPalCommerceScriptLoader {
                 'enable-funding': enableFunding.length > 0 ? enableFunding : undefined,
                 'disable-funding': disableFunding.length > 0 ? disableFunding : undefined,
                 commit,
-                components: ['buttons', 'hosted-fields', 'messages', 'payment-fields', 'legal'],
+                components: [
+                    'buttons',
+                    'hosted-fields',
+                    'messages',
+                    'payment-fields',
+                    'legal',
+                    ...googlePayComponent,
+                ],
```

Comment:
[optional] I think its ok to move google pay check here:

```suggestion
                components: [
                    'buttons',
                    'hosted-fields',
                    'messages',
                    'payment-fields',
                    'legal',
                    ...(isGooglePayEnabled ? ['googlepay'] : []),
                ],
```

Reply:
I left it as it was